tests/helpers/configs.ts

```diff
@@ -1,10 +1,13 @@
 import { Linter } from "eslint";
+import { join } from "path";
```

Comment:
Just a suggestion but when I import functions from **path**, I always tend to alias them so it's obvious what they do when used later on in the code.

So I would probably import **join** like this:

```ts
import { join as joinPaths } from "path";
```

Reply:
I went with `import * as path from "path";` for this. Since there is no need to treeshake named imports aren't really necessary.